dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/metrics/BatchMetricsDispatcher.kt

```diff
@@ -218,6 +235,15 @@ internal class BatchMetricsDispatcher(
         /* The value for the type of the metric.*/
         internal const val BATCH_CLOSED_TYPE_VALUE = "batch closed"
 
+        /* The value of the tracking consent according with this file origin.*/
+        internal const val TRACKING_CONSENT_KEY = "consent"
+
+        /* The file name.*/
+        internal const val FILE_NAME = "name"
+
+        /* The thread name from which the current metric was sent.*/
+        internal const val THREAD_NAME = "thread"
```

Comment:
how it can be useful? is it actionable attribute and what knowledge can it give?

Reply:
we decided that is better to have more and remove it later if not needed. This is for tracking any concurrency issue. We know that theoretically a `batch_close` should be sent with more or less same thread (depending of the delay between them as the executor could kill that thread and spawn a new one).

Reply:
so we are going to assert is 2 values between 2 metrics from the same source are equal or not? Not sure if it is possible in automatic way, but okay, fair enough (not still sure that it can help us).